.github/workflows/publish.yml

```diff
@@ -0,0 +1,31 @@
+name: Publish prerelease in NPM
+
+on:
+  workflow_dispatch:
+  workflow_run:
+    workflows: [Tests]
+    types: [completed]
+    branches: [develop, 'release/**']
```

Comment:
This makes the workflow run not only on the actual release branch but also the branch being a base for the docs PR during the release process, [as they're usually named `release/[version]-docs`](https://kb.handsontable.com/display/BOOK/Document+a+Handsontable+release).
I'm not sure if that's what we want.

This might be a good time to rethink the docs PR branch name, as `release/[version]-[something]` might be considered an alpha/beta/(...) release branch.

Reply:
If, in the docs release branch, there would be a change that triggers the tests, then yes the prerelease would be created. Otherwise, the workflow is skipped. IMO leaving this as is would be ok.

Reply:
🤷‍♂️ cc @warpech

Reply:
During the code freeze period, any change in the package files should be worth testing, so it should be worth having a pre-release.

However, in the `release/[version]-docs` branches, I think we are only supposed to make changes in JSDoc and sometimes they are done in many separate commits.

I agree that publishing pre-release packages that often might be confusing.

I recommend we set the workflow like this:

```
    branches:
      - 'develop'
      - 'release/**'
      - '!release/**-docs'
```

Source: https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions

Reply:
@warpech But wouldn't setting `'!release/**-**'` block "postfix" releases (like `13.0.0-alpha` etc) from being published?

Reply:
We can easily change the rule at any time if the above suggestion becomes is too limiting.